recipe/meta.yaml

```diff
@@ -20,16 +20,18 @@ requirements:
   host:
     - python >=3.7
     - pip
-    - setuptools_scm[toml] >=3.4
-    - setuptools_scm_git_archive >=1.0
+    - setuptools >=61.0.0
+    - setuptools_scm >=7.0.1
   run:
     - python >=3.7
-    - scipy >=1.4.1
-    - click >=7.0
+    - scipy >=1.2.0
+    - click >=8.0.0
     - tqdm >=4.56.0
-    - jsonschema >=3.0.0
+    - jsonschema >=4.15.0
     - jsonpatch >=1.15
     - pyyaml >=5.1
+    - importlib_resources >=1.4.0   # python < 3.9
+    - typing_extensions >=3.7.4.3   # python < 3.8
```

Comment:
In https://github.com/conda-forge/staged-recipes/pull/19648 it was mentioned that there doesn't seem to be a way to install `importlib_resources` only on Python < 3.9 https://github.com/conda-forge/staged-recipes/pull/19648#issuecomment-1223443300

@kratsg it is your understanding that we just need to install these on every Python version then, correct? Even when they aren't needed, unlike [how `pip` does it](https://github.com/scikit-hep/pyhf/blob/2d791e828202375932bbf778eefb2e5aa1c3a5d1/setup.cfg#L44-L45)?

@chrisburr can you confirm that?

Reply:
that seems to be the current way yeah... not the most symmetric but I'm ok with it for now if this is what `conda` prefers.

Reply:
Okay, I'll merge this for now given @kratsg's feedback.

@chrisburr, if you do have any general advice for the future on how to handle this better I'd love to hear it. :+1: